index.html

```diff
@@ -427,48 +427,204 @@ <h3>
+and to be aware of exposure of QR Codes during digital credential interactions.
+        </p>
+      </section>
+
+      <section>
+        <h3>Data Integrity</h3>
+
+        <p class="issue" title="Work in progress">
+Explain that the API does not provide data integrity on the digital
+credential requests or responses and that responsibility is up to the
+underlying protocol used for the request or response.
+        </p>
+      </section>
+
+      <section>
+        <h3>Authentication</h3>
+
+        <p class="issue" title="Work in progress">
+Explain that authentication (such as a PIN code to unlock) to a particular app,
+such as a digital wallet, that responds to an API request is crucial in
+high-risk use cases.
+        </p>
+      </section>
+
+      <section>
+        <h3>Cross-Site Scripting (XSS) and Cross-Site Request Forgery (CSRF)</h3>
+
+        <p class="issue" title="Work in progress">
+Explain what attacks are possible via XSS and CSRF, if any.
+        </p>
+      </section>
+
+      <section>
+        <h3>Session Security</h3>
+
+        <p class="issue" title="Work in progress">
+Explain that once a secure session is established at a website using
+credentials exchanged over this API, that the subsequent security is no
+longer a function of the credential used or this API and is up to the
+session management utilized on the website.
+        </p>
+      </section>
+
     </section>
+
     <section class="informative">
       <h2>
         Privacy Considerations
       </h2>
-      <p class="issue" title="Work in Progress">
+      <div class="issue" title="Privacy Considerations section is a work in progress">
+        <p>
         This section is a work in progress as this document evolves.
-      </p>
-      <p>
-        The documents listed below outline various privacy considerations for
-        Digital Credentials, both broadly and for presentation on the web.
-        Their contents will be integrated into this document gradually.
-      </p>
-      <ul>
-        <li>
-          <a href=
-          "https://github.com/WICG/digital-credentials/blob/main/horizontal-reviews/security-privacy.md">
-          TAG Security and Privacy Considerations Questionnaire (WIP)</a>
-        </li>
-        <li>
-          <a href="https://github.com/w3cping/credential-considerations">W3C
-          Privacy Interest Group (PING) Credentials Considerations
-          repository</a> which contains in-progress list of concerns and
-          implications of identity-verification and credential presentation,
-          for consideration in Internet and Web standardization
-        </li>
-      </ul>
+        </p>
+        <p>
+          The documents listed below outline various privacy considerations for
+          Digital Credentials, both broadly and for presentation on the web.
+          Their contents will be integrated into this document gradually.
+        </p>
+        <ul>
```

Comment:
Add the threat model here too? It's probably even more about privacy than security I'd say right @simoneonofri?

Reply:
@RByers, the generic one is a hybrid of security and privacy as per the frameworks used so that it can fit one or both

Reply:
Done in https://github.com/WICG/digital-credentials/pull/189/commits/2645588e428ef6441edf6d2b1c60378ef70e96d3.